### ISSUE TITLE
Remove or add a Deprecated comment to the RFG check

### ISSUE THREAD

**Issue by bonafideduck** [trailofbits/winchecksec#82]
According to [Windows 10 security: How the shadow stack will help to keep the hackers at bay](https://www.techrepublic.com/article/windows-10-security-how-the-shadow-stack-will-help-to-keep-the-hackers-at-bay/) published by TechRepublic in April of 2020, Return Flow Guard (RFG) was never released by Microsoft:

> But when the same team looked at how they could attack their own design, they found a race condition that meant some apps weren't protected and decided not to ship it at all.

Displaying this protection may cause people like me(1) to spend a lot of unneeded time trying to figure out how to enable this technology.  The above article is consistent with what I experienced.

Can the check be removed or at least have the documentation show a deprecated flag?

---
(1) I am Mark Eklund, an employee of Zoom Video Communications.

**Comment by woodruffw** [comment]
> Can the check be removed or at least have the documentation show a deprecated flag?

Yeah, adding a deprecated note sounds good to me. I'm happy to accept a PR that tweaks the description string for the RFG detection; otherwise, I'll get around to it when I have some free time 🙂
